tests/basic/storage_pool_test.go

```diff
@@ -5794,3 +5794,226 @@ func addNewPools(n node.Node, numPools int) error {
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get the Pool UUID on which IO is running
+		poolUUID, err := GetPoolIDWithIOs()
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+		log.FailOnError(err, "Failed to get pool using UUID [%v]", poolUUID)
+
+		nodeDetail, err := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(err, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+
+		PoolDetail := GetPoolsDetailsOnNode(*nodeDetail)
+		log.InfoD("Pool Details [%v] Picked for Delete", PoolDetail)
+
+		// invalidPoolID is total Pools present on the node + 1
+		invalidPoolID := fmt.Sprintf("%d", len(PoolDetail)+1)
+
+		log.InfoD("Bring Node to Maintenance Mode")
+		log.FailOnError(Inst().V.EnterMaintenance(*nodeDetail), fmt.Sprintf("Failed to bring Pool [%s] to Mainteinance Mode on Node [%s]", poolUUID, nodeDetail.Name))
+
+		log.InfoD("Wait for Node to Enter Maintenance Mode")
+		WaitTillEnterMaintenanceMode(*nodeDetail)
+
+		// Delete the Pool with Invalid Pool ID
+		err = Inst().V.DeletePool(*nodeDetail, invalidPoolID)
+		dash.VerifyFatal(err != nil, true, fmt.Sprintf("Pool Delete Status is SUCCESS. Expected Failure! : Node Detail [%v]", nodeDetail))
+		log.InfoD("Deleting Pool with InvalidID Errored as expected [%v]", err)
+
+		log.InfoD("Bring Node out of Maintenance Mode")
+		ExitFromMaintenanceMode(*nodeDetail)
+
+		JustAfterEach(func() {
+			defer EndTorpedoTest()
+			log.InfoD("Exit from Maintenance mode if Pool is still in Maintenance")
+			ExitNodesFromMaintenanceMode()
+			AfterEachTest(contexts, testrailID, runID)
+		})
+	})
+})
+
+var _ = Describe("{PoolResizeInvalidPoolID}", func() {
+	var testrailID = 79487
+	// Testrail Description : Resize with invalid pool ID
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/84470
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("PoolResizeInvalidPoolID",
+			"Resize with invalid pool ID",
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+	stepLog := "Resize with invalid pool ID"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("deleteinvalidpoolid-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get the Pool UUID on which IO is running
+		poolUUID, err := GetPoolIDWithIOs()
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+		log.FailOnError(err, "Failed to get pool using UUID [%v]", poolUUID)
+
+		nodeDetail, err := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(err, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+
+		// invalidPoolUUID Generation
+		id := uuid.New()
+		invalidPoolUUID := fmt.Sprintf("%s", id.String())
```

Comment:
nit: You don't need to `fmt.Sprintf("%s", id.String())`, `id.String()` is already a `string`

Reply:
fixed